dd-java-agent/agent-tooling/src/main/java/datadog/trace/agent/decorator/HttpClientDecorator.java

```diff
@@ -92,7 +92,8 @@ public AgentSpan onResponse(final AgentSpan span, final RESPONSE response) {
       if (status != null) {
         span.setTag(Tags.HTTP_STATUS.getKey(), status);
 
-        if (Config.get().getHttpClientErrorStatuses().contains(status)) {
+        if (Config.get().getHttpClientErrorStatuses().contains(status)
+            || Config.get().getHttpServerErrorStatuses().contains(status)) {
```

Comment:
Could you explain why you made this change?

Reply:
This replaces the `Status5XXDecorator` which was marking 5xx status codes as error before.

Reply:
hmm... I forgot that was still applying for clients.  That doesn't seem right to me.  Any objections to removing `Status5XXDecorator` without making this change?

Reply:
The current behavior makes sense to me (client spans being marked as error when they receive 5xx from server). This code is going to change in #12 anyways to match https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-http.md#status.

Reply:
We chatted about this and decided to revert this specific change and update the default config range.  This allows the error statuses to be configured independently as intended.